pythonforandroid/build.py

```diff
@@ -650,22 +653,22 @@ def build_recipes(build_order, python_modules, ctx):
             recipe.postbuild_arch(arch)
 
     info_main('# Installing pure Python modules')
-    run_pymodules_install(ctx, python_modules)
+    run_pymodules_install(
+        ctx, python_modules, project_dir,
+        ignore_setup_py=ignore_project_setup_py
+    )
 
     return
 
 
-def run_pymodules_install(ctx, modules):
+def run_pymodules_install(ctx, modules, project_dir, ignore_setup_py=False):
     modules = list(filter(ctx.not_has_package, modules))
 
-    if not modules:
-        info('There are no Python modules to install, skipping')
-        return
-
-    info('The requirements ({}) don\'t have recipes, attempting to install '
-         'them with pip'.format(', '.join(modules)))
-    info('If this fails, it may mean that the module has compiled '
-         'components and needs a recipe.')
+    if len(modules) > 0:
```

Comment:
Minor: I think the style guide actually suggest to use `if not module:` like it was before to check empty list, see https://www.python.org/dev/peps/pep-0008/